src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -440,6 +468,9 @@ public Problem copy(String newDisplayName) {
             clone.addGroup(group);
         }
         
+        clone.setSandboxType(this.getSandboxType());
+        clone.setMemoryLimit(this.getMemoryLimitMB());
```

Comment:
See comment below regarding the name of this setter.

Reply:
Changed in [this commit](https://github.com/pc2ccs/pc2v9/pull/732/commits/ac45863596ea4b46d29a95e70db66998c8b423d5)